tests/forms_tests/tests/test_forms.py

```diff
@@ -746,6 +746,13 @@ class BeatleForm(Form):
             [str(bf[1]), str(bf[2]), str(bf[3])],
         )
 
+    def test_boundfield_bool(self):
+        # A BoundField without any choices (i.e. subwidgets) evaluates to True.
```

Comment:
I don't care either way but existing tests in this file are using comments and a docstring is going to take 3 lines rather than 1.